pipeline-runner/R/gem2s-6-prepare_experiment.R

```diff
@@ -22,6 +22,12 @@ prepare_experiment <- function(input, pipeline_config, prev_out) {
 
   message("Merging Seurat Objects...")
   scdata <- merge_scdatas(scdata_list)
+
+  #If subsetting all cells, Seurat will not reorder the cells in the object. We need to subset to [-1] and [1] and merge to shuffle.
+  set.seed(42)
+  shuffle_mask <- sample(colnames(scdata),ncol(scdata))
+  scdata <- merge(scdata[,shuffle_mask[-1]],scdata[,shuffle_mask[1]])
```

Comment:
```suggestion
  scdata <- merge(scdata[,shuffle_mask[1]],scdata[,shuffle_mask[-1]])
```

so that `shuffle_mask` and `scdata` have same order